sprouts.py

```diff
@@ -0,0 +1,246 @@
+import pygame
+from pygame import *
+import sys
+import time
+import random
+import math
+
+#Contains Game Settings: sproutset.py on Github
+from sproutset import Settings
+
+####OBJECTS - Will move later to .py or class
+#Game Window 
+pygame.init()
+ss = Settings()
+win = pygame.display.set_mode((ss.width, ss.height))
+pygame.display.set_caption('The Sprouts Game')
+
+running = True
+clock = pygame.time.Clock()
+
+#Player
+PlayerOne = 'Test1'
+PlayerOneScore = 0
+
+PlayerTwo = 'Test2'
+PlayerTwoScore = 0
+
+turn = 1 
+NumberStartingDots = 3
+
+#Dots
+dot_pos = []
+
+#Lines
+all_lines = []
+current_line = []
+
+####SPROUTS
+def sprouts():
+    global ss
+    #Start Screen
+    win.fill(ss.WHITE, None, 0)
+    text_writing('The Sprouts Game', ss.BLACK, 500, 25, 25) #Textwriting function
+    text_writing(f'{PlayerOne} score =' + str(PlayerOneScore), ss.BLACK, 125, 20, 12) #Updating Player Score text objects
+    text_writing(f'{PlayerTwo} score =' + str(PlayerTwoScore), ss.BLACK, 125, 40, 12)
+    pygame.draw.line(win, ss.BLACK, (0,52), (1000,52)) #Used to mimic an above playing field menu
+    pygame.display.update()
+
+    #Instructions screen 
+    #redraw_box()
+    #text_writing('The Rules:\n1. Not more than 3 lines per dot. Ect.', ss.BLACK, 500, 300, 25)    
+    
+    start_sequence() #Function that draws three random starter dots
+    
+    #Game Modes
+    global current_line
+    LineMode = True
+    DrawingLine = False
+    DotMode = False
+
+    ####MAIN GAME LOOP:
+    while True:
+
+        #Starts collecting key and mouse events:
+        for event in pygame.event.get():
+            if event.type == pygame.QUIT: #Press 'x' to exit
+                sys.exit()
+
```

Comment:
I don't know whether it is my computer. But it seems not to close the window. 
I tried to do it this way and it seems to work with this code:
    running = True
    ####MAIN GAME LOOP:
    while running:

        #Starts collecting key and mouse events:
        for event in pygame.event.get():
            if event.type == pygame.QUIT: #Press 'x' to exit
                running = False
                pygame.quit()
                sys.exit()

Reply:
Good to know! Changed it to this